src/test/java/games/strategy/util/MatchTest.java

```diff
@@ -74,11 +75,11 @@ public void testAnyMatch() {
 
   @Test
   public void testNoneMatch() {
```

Comment:
Nit: IMO, this is testing the implementation of `Stream#noneMatch()`, which I don't believe is necessary, and this method should be removed.  If the intention was to prove that `Match` can now be used as a `Predicate`, I think you could simply make a copy of `testMatch()` and exercise the `Match#test()` method directly:

```diff
diff --git a/src/test/java/games/strategy/util/MatchTest.java b/src/test/java/games/strategy/util/MatchTest.java
index b47ca0e81..bb26abbe2 100644
--- a/src/test/java/games/strategy/util/MatchTest.java
+++ b/src/test/java/games/strategy/util/MatchTest.java
@@ -4,7 +4,6 @@ import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.util.Arrays;
-import java.util.stream.Stream;
 
 import org.junit.jupiter.api.Test;
 
@@ -23,6 +22,13 @@ public class MatchTest {
   }
 
   @Test
+  public void testTest() {
+    assertFalse(IS_ZERO_MATCH.test(-1));
+    assertTrue(IS_ZERO_MATCH.test(0));
+    assertFalse(IS_ZERO_MATCH.test(1));
+  }
+
+  @Test
   public void testInverse() {
     assertFalse(Matches.always().invert().match(VALUE));
     assertTrue(Matches.never().invert().match(VALUE));
@@ -74,15 +80,6 @@ public class MatchTest {
   }
 
   @Test
-  public void testNoneMatch() {
-    assertTrue(Stream.<Integer>empty().noneMatch(IS_ZERO_MATCH), "empty collection");
-    assertTrue(Arrays.asList(-1).stream().noneMatch(IS_ZERO_MATCH), "none match (one element)");
-    assertTrue(Arrays.asList(-1, 1).stream().noneMatch(IS_ZERO_MATCH), "none match (multiple elements)");
-    assertFalse(Arrays.asList(-1, 0, 1).stream().noneMatch(IS_ZERO_MATCH), "some match");
-    assertFalse(Arrays.asList(0, 0, 0).stream().noneMatch(IS_ZERO_MATCH), "all match");
-  }
-
-  @Test
   public void testBuildAll() {
     assertTrue(Match.newCompositeBuilder().all().match(VALUE));

```

With the removal of `testNoneMatch()`, that will also keep test coverage unchanged.